include/verilated_vpi.cpp

```diff
@@ -631,16 +631,14 @@ class VerilatedVpiCbHolder final {
 };
 
 class VerilatedVpiPutHolder final {
-    const VerilatedVar* m_varp;
-    const VerilatedScope* m_scopep;
+    VerilatedVpioVar m_vo;
```

Comment:
Maybe still v_varp?  m_vo seems a little too short to be easily readable and understood.